src/mmw/apps/modeling/migrations/0020_remove_project_is_activity.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modeling', '0019_project_gis_data'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
I don't think we need to remove it, but we may consider applying this migration after the 1.16.0 release is live.

For a brief period, the previous release may have to interact with the new database schema as we apply migrations before cutting over completely to the new release. The other approach is to cut over to the new release, and then apply migration (implying that the code in 1.16.0 would work OK with the current schema).

Given that we're removing a field, it seems like applying the relation after 1.16.0 is deployed seems like the way to go. Agreed?